docs_src/design/ucr/Device-Parent-Child-Relationships.md

```diff
@@ -3,10 +3,11 @@ This UCR describes Use Cases for new Device metadata for Parent to Child Relatio
 
 ### Submitters
 - Tom Brennan (Eaton)
+- Corey Mutter (Eaton)
 
 ## Change Log
-- [pending](https://github.com/edgexfoundry/edgex-docs/pull/800) (2022-08-22)
```

Comment:
Need a line for Approved linked to this PR (I guess)

Reply:
Done